Dockerfile

```diff
@@ -8,4 +8,4 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' \
         -o /bin/a.out ./cmd/cloudshell_open
 
 FROM gcr.io/cloudshell-images/cloudshell:latest
-COPY --from=build /bin/a.out /bin/cloudshell_open
\ No newline at end of file
+COPY --from=build /bin/a.out /bin/cloudshell_open
```

Comment:
**Question** (mostly for my personal understanding, no action required): Does this image need to be manually built and pushed after merging this pull-request?

Reply:
Not manual; there's a Cloud Build Trigger set to build on merge to main for this repo.